packages/@guardian/source-react-components/src/button/types.ts

```diff
@@ -32,4 +32,9 @@ export interface SharedButtonProps extends Props {
 	 * When set, the icon (if visible) will move slightly to the right on hover.
 	 */
 	nudgeIcon?: boolean;
+	/**
+	 * When true, a loading spinner will appear to the right of the button content.
```

Comment:
Doesn't it appear to the left by default? (Sorry, this is a super minor comment 😅)

Reply:
changed this now :)